.gitmodules

```diff
@@ -3,7 +3,7 @@
 	url = https://github.com/nlohmann/json
 [submodule "other/websocketpp"]
 	path = other/websocketpp
-	url = https://github.com/zaphoyd/websocketpp
+	url = https://github.com/Vasyl-Baran/websocketpp_logi
```

Comment:
This should be pointing to a Logitech repository.

https://github.com/Logitech/logi_websocketpp

Reply:
nice catch! I'm just fixed it, could you please take another look? Thank you!